src/jit/emitxarch.cpp

```diff
@@ -1969,6 +1969,7 @@ inline UNATIVE_OFFSET emitter::emitInsSizeSV(code_t code, int var, int dsp)
 
 inline UNATIVE_OFFSET emitter::emitInsSizeSV(instrDesc* id, code_t code, int var, int dsp)
 {
+    assert(id->idIns() != INS_invalid);
```

Comment:
There was a few places passing in `id` without initializing `Ins` first, I added an assert to the various `emitInsSize*` functions that took a `instrDesc`